README.md

```diff
@@ -145,26 +144,27 @@ Open-source clients to the plotly.js APIs are available at these links:
 
 |   | GitHub repo | Getting started |
 |---|--------|---------|
-|**R / RStudio**| [ropensci/plotly](https://github.com/ropensci/plotly) | [plotly/r/getting-started](https://plotly.com/r/getting-started) |
 |**Python / Pandas / IPython notebook**| [plotly/plotly.py](https://github.com/plotly/plotly.py) | [plotly/python/getting-started](https://plotly.com/python/getting-started) |
+|**R / RStudio**| [ropensci/plotly](https://github.com/ropensci/plotly) | [plotly/r/getting-started](https://plotly.com/r/getting-started) |
 |**MATLAB**| [plotly/matlab-api](https://github.com/plotly/matlab-api) | [plotly/matlab/getting-started](https://plotly.com/matlab/getting-started) |
 |**node.js / Tonicdev / Jupyter notebook**| [plotly/plotly-notebook-js](https://github.com/plotly/plotly-notebook-js) | |
 
-## Creators
+## Maintainers
```

Comment:
Let's maintain `Creators` in the title here. 
What about `Creators & maintainers`?

Reply:
Why the attachment to "creators" ? I don't love that that excludes community contributors... Maintainer at least is a fairly formal role

Reply:
There are a number of people in the list below (including Plotly contractors in the Hall of Fame) who were not in charge of "maintaining" `plotly.js`; however their contribution was remarkable.

In respect to community contributors I suggest we add the link below to the bottom of list: https://github.com/plotly/plotly.js/graphs/contributors.

Reply:
How about Maintainers for the top list and Notable Contributors for the bottom list? I don't love that Jon Mease isn't in the "Hall of Fame" for example... he did add a whole trace type :)

Reply:
We should definitely add Jon Mease to the top list as an active contributor not the bottom list.